<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="apples-MacBook-Pro-4.local" name="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY" tests="28" failures="2" timestamp="15 Jul 2019 12:24:29 GMT" time="254.921" errors="1">
  <testcase name="gettingDashboard" time="0.271" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="0.250" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="12.718" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.644" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.495" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.333" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="11.915" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.554" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.359" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="12.749" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.645" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="6.933" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:101)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:45)
at com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY.gettingDashboard(Test_APAC_iOS_Frequent_Payment_External_FCY_LCY.java:25)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="43.558" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY">
    <error type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for presence of element located by: By.xpath: //XCUIElementTypeStaticText[@name=&quot;SET RECURRING&quot;] (tried for 20 second(s) with 500 milliseconds interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for presence of element located by: By.xpath: //XCUIElementTypeStaticText[@name="SET RECURRING"] (tried for 20 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:113)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:283)
at com.citi.mobileAutomation.reusableFunction.ResuableFunction.getTextFromElementByWait(ResuableFunction.java:204)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:47)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:45)
at com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY.gettingDashboard(Test_APAC_iOS_Frequent_Payment_External_FCY_LCY.java:25)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.xpath: //XCUIElementTypeStaticText[@name="SET RECURRING"]
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'apples-MacBook-Pro-4.local', ip: '192.168.100.24', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.4', java.version: '1.8.0_211'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.ExpectedConditions.lambda$findElement$0(ExpectedConditions.java:896)
at java.util.Optional.orElseThrow(Optional.java:290)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:895)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:44)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:183)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:180)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:260)
... 28 more
]]>
    </error>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="59.379" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.537" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="3.286" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.949" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.611" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.753" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="0.201" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="6.508" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="5.677" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.499" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="13.520" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="41.621" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:101)
at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:45)
at com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY.gettingDashboard(Test_APAC_iOS_Frequent_Payment_External_FCY_LCY.java:25)
at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="2.317" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.225" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
  <testcase name="gettingDashboard" time="5.414" classname="com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY"/>
</testsuite> <!-- com.citi.mobileAutomation.test_SG.Test_APAC_iOS_Frequent_Payment_External_FCY_LCY -->
